packages/eslint-config-ckeditor5/.eslintrc.js

```diff
@@ -315,12 +315,156 @@ module.exports = {
 		'mocha/no-sibling-hooks': 'error',
 		'mocha/no-top-level-hooks': 'error'
 	},
-	'overrides': [
+	overrides: [
 		{
-			'files': [ '**/tests/**/*.js' ],
-			'rules': {
+			files: [ '**/*.ts' ],
+			plugins: [
+				'@typescript-eslint',
+				'ckeditor5-rules',
+				'mocha'
+			],
+			parser: '@typescript-eslint/parser',
+			extends: [
+				'eslint:recommended',
+				'plugin:@typescript-eslint/eslint-recommended',
+				'plugin:@typescript-eslint/recommended'
+			],
+			rules: {
+				'@typescript-eslint/ban-types': [
+					'error',
+					{
+						types: { Function: false },
+						extendDefaults: true
+					}
+				],
+
+				'@typescript-eslint/consistent-type-assertions': [
+					'error',
+					{
+						assertionStyle: 'as',
+						objectLiteralTypeAssertions: 'allow-as-parameter'
+					}
+				],
+
+				'@typescript-eslint/consistent-type-imports': 'error',
```

Comment:
Does it block the `import type ...` expression?

Reply:
It requires it.